airbyte-integrations/connectors/source-dynamodb/src/main/java/io/airbyte/integrations/source/dynamodb/DynamodbUtils.java

```diff
@@ -27,10 +32,16 @@ private DynamodbUtils() {
 
   public static DynamoDbClient createDynamoDbClient(final DynamodbConfig dynamodbConfig) {
     final var dynamoDbClientBuilder = DynamoDbClient.builder();
+    AwsCredentialsProvider awsCredentialsProvider;
+    if (!StringUtils.isBlank(dynamodbConfig.accessKey()) && !StringUtils.isBlank(dynamodbConfig.secretKey())) {
+      AwsCredentials awsCreds = AwsBasicCredentials.create(dynamodbConfig.accessKey(), dynamodbConfig.secretKey());
```

Comment:
What about adding some logs here:
`Creating credentials using access key and secret key`

and `Using Role Based Access`